src/Credentials.js

```diff
@@ -212,23 +212,18 @@ module.exports = class Credentials {
     return this._bearer_token_promise;
   }
 
-  async authorizationHeader(url) {
+  async authorizationHeader(url, method, body) {
```

Comment:
Please also update [the corresponding typescript definition](https://github.com/HunterLarco/twitter-v2/blob/master/src/Credentials.d.ts#L30)

Reply:
I haven't coded in ts before, sorry 😥